packages/site/src/ui/prop-types.js

```diff
@@ -1,57 +1,114 @@
-import Badge from '@pluralsight/ps-design-system-badge'
-import * as core from '@pluralsight/ps-design-system-core'
 import PropTypes from 'prop-types'
 import React from 'react'
+
+import Badge from '@pluralsight/ps-design-system-badge'
+import {
+  colorsBlue,
+  colorsPink,
+  layout
+} from '@pluralsight/ps-design-system-core'
 import Table from '@pluralsight/ps-design-system-table'
 
 import { Heading, SectionHeading } from './index.js'
 
-const BadgeSpacer = () => <span style={{ width: core.layout.spacingXSmall }} />
+export default function PropTypesTable(props) {
+  const { props: propsTable, title = 'PropTypes' } = props
 
-const union = obj => <code>{Object.keys(obj).join(' | ')}</code>
+  return (
+    <>
+      <div className="proptypes">
+        <SectionHeading>{title}</SectionHeading>
 
-const row = ([name, value, required, defaultValue, description]) => ({
-  name,
-  value,
-  required,
-  defaultValue,
-  description
-})
+        <Table className="table">
+          {Array.isArray(propsTable)
```

Comment:
Didn't want to add "make proptypes api not terrible" too? :)

Reply:
i get too neurotic with organizing and formatting things. it's a legit problem sometimes 😬